lib/utils.pm

```diff
@@ -1113,7 +1113,13 @@ sub set_hostname {
                 next if ($dev eq 'lo');
                 next if !($line =~ /connected/);
 
-                assert_script_run 'nmcli device disconnect ' . $dev;
+                if (get_var('QEMU_NO_KVM')) {
+                    # Default timeout (10 seconds) may be too short with qemu NO kvm, so increase to 20s - poo#131366
+                    assert_script_run 'nmcli -w 20 device disconnect ' . $dev;
+                }
+                else {
+                    assert_script_run 'nmcli device disconnect ' . $dev;
+                }
```

Comment:
That's rather ugly and hard for people to follow in all the different cases. How about simply unconditionally

```suggestion
                assert_script_run 'nmcli -w 60 device disconnect ' . $dev;
```

Reply:
I suspect people would prefer to catch if default timeout is not enough in a regular case.